internal/kafka/consumer.go

```diff
@@ -12,9 +12,17 @@ import (
 	"github.com/redhatinsights/ros-ocp-backend/internal/logging"
 )
 
-func StartConsumer(kafka_topic string, handler func(msg *kafka.Message)) {
+func StartConsumer(kafka_topic string, handler func(msg *kafka.Message, consumer_object *kafka.Consumer), auto_commit_option ...bool) {
```

Comment:
Instead of changing function signature do you think it would be easy to have config variable which can be tweak by setting environment variable. Default value of this variable can be set to true and then on line 39 and 48 we can do something like `"enable.auto.commit": cfg.AUTOCOMMIT `

Reply:
We could go with config option, but updating the function signature would help us with reusing the consumer function programmatically.

In this specific case the consumer object can be passed onto the handler, so in future we may want this for some handler-specific kafka functionality for instance`consumer_object.CommitMsg`.